pkg/odin/instance_test.go

```diff
@@ -1,9 +1,75 @@
 package odin
 
 import(
+	"fmt"
 	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
 )
 
+func getTime(original string) (parsed time.Time) {
+	parsed, _ = time.Parse(
+		RFC8601,
+		original,
+	)
+	return
+}
+
+type mockRDSClient struct {
+	rdsiface.RDSAPI
+	dbInstances []*rds.DBInstance
+	dbSnapshots []*rds.DBSnapshot
+}
```

Comment:
I had to duplicate the mockRDSClient because it's defined in `odin_test` package. We should move this to `internal/test/mockRDSClient` or similar, and import it for both testing packages.